src/main/java/org/jabref/logic/formatter/bibtexfields/NormalizePagesFormatter.java

```diff
@@ -63,7 +64,8 @@ public String format(String value) {
         // Remove pages prefix
         String cleanValue = value.replace("pp.", "").replace("p.", "");
         // remove unwanted literals including en dash, em dash, and whitespace
-        cleanValue = cleanValue.replaceAll("\u2013|\u2014", "-").replaceAll(REJECT_LITERALS, "");
+        cleanValue = EM_EN_DASH_PATTERN.matcher(cleanValue).replaceAll("-");
```

Comment:
I would put this into the `PAGES_DETECT_PATTERN`.

Reply:
No, it makes sense to first replace all occurrences of en/em dashes before using pages detect pattern with a minus.

Reply:
@Siedlerchr In BibTeX, an em-dash is ---, and an en-dash is --. Maybe the replacement should enforce this?

Reply:
In pages you can either have a minus or a -- https://www.bibtex.com/f/pages-field/ at least that's what the mods exporter expects

Reply:
The URL you give mentions "It is standard to use an en-dash (“--”) to separate a page range." 😉

Reply:
> No, it makes sense to first replace all occurrences of en/em dashes before using pages detect pattern with a minus.

What are the advantages of this approach? I only see that the performance is worse as you apply to regex per string instead of only one.

I also agree with mlep, that `--` is the desired format (and that also the format that the formatter produces). But this already covered by the part `?:-{1,2}` in the regex.

Reply:
I looked a bit into it, the problem is that for the test case R404-R405 the page detect pattern regex does not match anything non-numeric and therefore would return the orginal value.

Reply:
I would suggest we weaken the regex to follow the biblatex def: (2.2.1)
>A range is somethingoptionally followed by one or more dashes optionally followed by some non-dash (e.g.5--7).